packages/core/src/components/ui/Dropdown/_Dropdown.scss

```diff
@@ -132,6 +132,7 @@ $ufx-dropdown-input-border: 1px solid $light-gray1 !default;
       right: $ufx-dropdown-padding * 0.5;
       height: 100%;
       color: $ufx-dropdown-text-color;
+      margin: 0;
```

Comment:
this may affect layout at lot of places where Dropdown is used inside ufx-ui/* comps, could you customise styles inside bfx-hf-ui if required only for hf-ui layout

Reply:
![Снимок экрана от 2021-06-21 16-30-46](https://user-images.githubusercontent.com/81973123/122770411-1564c480-d2ae-11eb-839b-55a038674a65.png)
it is needed for all search inputs with cancel icons